indico/modules/rb_new/client/js/components/RoomBasicDetails.module.scss

```diff
@@ -15,9 +15,67 @@
  * along with Indico; if not, see <http://www.gnu.org/licenses/>.
  */
 
-.equipment-box {
+@import 'rb_new:styles/palette';
+
+.feature-box {
+    position: absolute;
+    display: flex;
+    bottom: -1px;
+    width: 100%;
+
+    .feature-entry {
+        background-color: $white;
+        border-radius: 3px 3px 0 0;
+        border-color: $pastel-gray $pastel-gray $white $pastel-gray;
+        border-width: 1px 1px 2px 1px;
```

Comment:
I think `border-width: 1px 1px 2px;` would suffice.

Reply:
dunno.. i never liked the 3-value style very much. with 1/2/4 values it's clear what it does, but with 3? at least i'd have to check the docs to see if it's `1 1 2 1` or `1 1 2 2`

Reply:
I'll vote to keep it as it is too. I'm not a fan of the 3-value notation either.